docs/performance_guide.md

```diff
@@ -9,6 +9,14 @@ There are many tricks to optimize PyTorch models for production including but no
 
 In general it's hard to optimize models and the easiest approach can be exporting to some runtime like ORT, TensorRT, IPEX or FasterTransformer. We have many examples for how to integrate these runtimes on the [TorchServe github page](https://github.com/pytorch/serve/tree/master/examples). If your favorite runtime is not supported please feel free to open a PR.
 
+<h4>torch.compile</h4>
+
+Starting with PyTorch 2.0, `torch.compile` provides out of the box speed up ( ~1.8x) for a large number of models. You can refer to this [dashboard](https://hud.pytorch.org/benchmark/compilers) which tracks this on a nightly basis.
+
+Models which have been fully optimized with `torch.compile` show performance improvements up to 10x
+
+You can find all the examples of `torch.compile` with TorchServe [here](../examples/pt2/)
```

Comment:
I'm not sure this will render correctly on the website, you'll need a full link

Reply:
done